packages/project/src/utils/ErrorCollector.ts

```diff
@@ -0,0 +1,12 @@
+export class ErrorCollector {
```

Comment:
Rather than moving this class to this package, can you just pass it in as an argument? This class will be rewritten by something I'm working on at which point we could change that. Until then I'd like to avoid duplicating this class definition. If you need a type you can just define an interface here (or just use any for now)

Reply:
moving since our most recent discussion here: https://norinauts.slack.com/archives/D01UH2VLGN8/p1627670548003900